### PR TITLE
Api check

### PR DESCRIPTION
**Pull request by abcdefghijohn**
Description:  
* Adds facade to call api 
* Adds poro to make api calls into objects
* Update route
* Serializer implemented
* Poro, Facade, Service tests built and pass

### CODE REVIEW

**Review by hopesgit — APPROVED** [review]
Looking good!

File: app/controllers/api/v1/yelp/businesses_search_controller.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-    conn = Faraday.new(url: 'https://relocate-08-yelp-microservice.herokuapp.com')
-    response = conn.get("/")
-    parsed = JSON.parse(response.body, symbolize_names: true)
-    render json: parsed
+  def utilities
+    data = BusinessesSearchFacade.utilities_search(params[:location], params[:type])
+    render json: UtilitySerializer.new(data)
+    # x = data.each_with_object([]) do |obj, array|
+    #   array << UtilitySerializer.new(obj)
+    # end
```
Comment (Stearnzy) [inline app/controllers/api/v1/yelp/businesses_search_controller.rb]: Hi! Why is this commented out for the merge to main?

File: app/poros/utility.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    @closed_bool = result[:is_closed]
+    @phone = result[:phone]
+    @distance = result[:distance]
+    @location = address(result[:location])
+    @id = result[:id]
+  end
+
+  def address(location)
+    "#{location[:address1]} #{location[:address2]}, #{location[:city]}, #{location[:state]}, #{location[:zip_code]}"
+  end
```
Comment (Stearnzy) [inline app/poros/utility.rb:22]: Thanks for interpolating this for FE!

**Review by Stearnzy — COMMENTED** [review]
Great job on testing format everyone!  Before approving, I am curious on that commented block of code in the businesses search controller..

**Review by Oxalisviolacea — APPROVED** [review]

File: app/facades/businesses_search_facade.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class BusinessesSearchFacade
+  def self.utilities_search(location, type)
+    utilities = BusinessesSearchService.utility_search(location, type)
+    open_businesses = utilities[:data].select {|result| result[:is_closed] == false}
```
Comment (Callbritton) [inline app/facades/businesses_search_facade.rb:4]: Excellent job on filtering the closed businesses!

**Review by Stearnzy — APPROVED** [review]

**Pull request merged**